src/screens/OtherProfile/index.js

```diff
@@ -289,7 +291,7 @@ const OtherProfile = () => {
 
     const __renderFollowingButton = () => {
       if (user_id === dataMain.user_id) return <></>;
-      if (dataMain.is_following)
+      if (dataMain.is_me_following_target)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `dataMain.is_following` to `dataMain.is_me_following_target` appears to be correct, as it now checks the new state variable `is_me_following_target` which should accurately reflect whether the current user is following the target user.

```diff
-      if (dataMain.is_following)
+      if (dataMain.is_me_following_target)
         return (
           <GlobalButton buttonStyle={{paddingLeft: 0}} onPress={() => handleSetUnFollow()}>
             <View style={styles.buttonFollowing(isAnonimity)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->